backend/src/main/kotlin/fr/gouv/cnsp/monitorfish/infrastructure/database/entities/ManualPriorNotificationEntity.kt

```diff
@@ -0,0 +1,163 @@
+package fr.gouv.cnsp.monitorfish.infrastructure.database.entities
+
+import fr.gouv.cnsp.monitorfish.domain.entities.logbook.*
+import fr.gouv.cnsp.monitorfish.domain.entities.logbook.messages.PNO
+import fr.gouv.cnsp.monitorfish.domain.entities.prior_notification.PriorNotification
+import fr.gouv.cnsp.monitorfish.domain.entities.vessel.UNKNOWN_VESSEL
+import fr.gouv.cnsp.monitorfish.domain.exceptions.BackendInternalException
+import io.hypersistence.utils.hibernate.type.json.JsonBinaryType
+import jakarta.persistence.*
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.Type
+import org.hibernate.annotations.UpdateTimestamp
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+
+@Entity
+@Table(name = "manual_prior_notifications")
+data class ManualPriorNotificationEntity(
+    @Id
+    @Column(name = "report_id", updatable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val reportId: String?,
+
+    @Column(name = "author_trigram")
+    val authorTrigram: String,
+
+    @Column(name = "cfr")
+    val cfr: String,
+
+    @Column(name = "created_at", insertable = false, updatable = false)
+    @CreationTimestamp
+    val createdAt: ZonedDateTime? = null,
+
+    @Column(name = "did_not_fish_after_zero_notice")
+    val didNotFishAfterZeroNotice: Boolean,
+
+    // ISO Alpha-3 country code
+    @Column(name = "flag_state")
+    val flagState: String?,
```

Comment:
```suggestion
    @Convert(converter = CountryCodeConverter::class)
    val flagState: CountryCode,
```
On essaie d'en mettre un peu partout, pour éviter de balader des magic string

Reply:
Oui je suis bien d'accord, le pire est que je l'ai fait ailleurs. Par contre j'ai remarqué qu'on mixait les 2 letters codes avec les 3 letters, ça ne pose pas de soucis ?

Reply:
En testant je comprends pourquoi je ne l'ai pas fait, ça fait planter un paquet de tests. Je ne vais pas avoir le temps d'améliorer ça sur cette PR. Il faudrait faire une PR de refacto qui remplace ça partout.

Reply:
#3279